modules/@amperka/power-control.js

```diff
@@ -0,0 +1,108 @@
+var Power = function(pin) {
+  this._pin = pin;
+  this._on = false;
+  this._brightness = 1.0;
```

Comment:
Not sure `_brightness` is a suitable name here. Generic `_duty` make a little more sense.